### PR TITLE
[Bug][dolphinscheduler-datasource-hive]Add hive-shims

### PR DESCRIPTION
**Pull request by jon-qj**
#7013 [Bug] [dolphinscheduler-datasource-hive] Using hive jdbc requires hive-shims jars

### CODE REVIEW

**Comment by kezhenxu94** [comment]
You need to update the license file, and known-dependencies.txt file

**Comment by jon-qj** [comment]
> You need to update the license file, and known-dependencies.txt file

Get it.Thx.

File: dolphinscheduler-dist/release-docs/licenses/LICENSE-hive-shims-0.23.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (kezhenxu94) [inline dolphinscheduler-dist/release-docs/licenses/LICENSE-hive-shims-0.23.txt]: These are standard Apache 2.0 so we don't need these copies, please modify https://github.com/apache/dolphinscheduler/blob/a8aed0f5f7d0c8fd2d8acf04359c9cff8276df14/dolphinscheduler-dist/release-docs/LICENSE#L211-L225

File: dolphinscheduler-dist/release-docs/LICENSE [hunk 2]
```diff
@@ -403,6 +403,13 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
...
     prometheus client_java(simpleclient) 0.12.0: https://github.com/prometheus/client_java, Apache 2.0
     snowflake snowflake-2010: https://github.com/twitter-archive/snowflake/tree/snowflake-2010, Apache 2.0
+    hive-shims 2.1.0: https://mvnrepository.com/artifact/org.apache.hive/hive-shims/2.1.0, Apache 2.0
+    hive-shims-0.23 2.1.0: https://mvnrepository.com/artifact/org.apache.hive.shims/hive-shims-0.23/2.1.0, Apache 2.0
+    hive-shims-common 2.1.0: https://mvnrepository.com/artifact/org.apache.hive.shims/hive-shims-common/2.1.0, Apache 2.0
+    hive-shims-scheduler 2.1.0: https://mvnrepository.com/artifact/org.apache.hive.shims/hive-shims-scheduler/2.1.0, Apache 2.0
+    hadoop-yarn-server-applicationhistoryservice 2.6.0: https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-yarn-server-applicationhistoryservice/2.6.0, Apache 2.0
+    hadoop-yarn-server-resourcemanager 2.6.0: https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-yarn-server-resourcemanager/2.6.0, Apache 2.0
+    hadoop-yarn-server-web-proxy 2.6.0: https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-yarn-server-web-proxy/2.6.0, Apache 2.0
```
Comment (kezhenxu94) [inline dolphinscheduler-dist/release-docs/LICENSE:414]: This is not in known/dependencies.txt?

Comment (jon-qj) [reply dolphinscheduler-dist/release-docs/LICENSE:414]: > 
Yep, and I've added them.

**Comment by CalvinKirs** [comment]
I don't know if there is a problem with this, because I see that it is excluded from the history, maybe you can add it manually. Does anyone else understand the reason for this?

**Pull request closed without merging**